### ISSUE TITLE
Tensorboard accuracy is not the same as the one displayed by the script

### ISSUE THREAD

**Issue by vladoski** [timesler/facenet-pytorch#141]
Hi, I'm trying to use this network to train it on my dataset.  
I have a little problem, Tensorboard's acc graph is quite odd. It goes to 1 in training accuracy even if I can't really see that much in the script's output, while finetuning the net.
Has anyone had this problem before?  
Thanks.

**Comment by timesler** [comment]
This is due to the fact that the tensorboard logging for the training pass logs the actual batch metrics and not the running average of those metrics.

**Comment by vladoski** [comment]
Okay, understood. Thanks!
